core-devel/linux+api/spec

```diff
@@ -1,8 +1,9 @@
 VER=6.12.4
+REL=1
 # RC=
 # Use this for RC releases.
 #SRCS="tbl::https://git.kernel.org/torvalds/t/linux-${VER%%.0}-rc${RC}.tar.gz"
 # Use this for stable releases.
 SRCS="tbl::https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-${VER%%.0}.tar.xz"
 CHKSUMS="sha256::6f35f821433d8421be7167990747c7c4a0c451958fb96883446301af13d71152"
-CHKUPDATE="anitya::id=6501"
+CHKUPDATE="anitya::id=375621"
```

Comment:
Maybe add a comment mentioning this anitya ID tracks a certain LTS branch instead of just the project "linux".

Reply:
Done.